### PR TITLE
Pioneer 18 SM deployment updates

### PR DESCRIPTION
**Pull request by rtravis-whoi**
P18 SM deployments; recovery info pending.
Reference for nominal depths - CGSN_Mooring_Component_Depths.xlsx attached
[CGSN_Mooring_Component_Depths.xlsx](https://github.com/WHOIGit/ooicgsn-asset-management/files/8483593/CGSN_Mooring_Component_Depths.xlsx)

### CODE REVIEW

**Comment by rtravis-whoi** [comment]
[Pioneer 18 AM Updates [DT-152]](https://trello.com/c/zFi7BfCv/152-pioneer-18-am-updates-dt-152)

File: deployment/CP01CNSM_Deploy.csv [hunk 1]
```diff
@@ -475,71 +475,71 @@ AR52,,AR61,,CP01CNSM-SBD12-04-PCO2AA000,14,1,2021-03-31T15:36:00,2021-11-03T17:0
...
+AR66,,,,CP01CNSM-SBD11-01-MOPAK0000,16,1,2022-04-10T14:35:00,,CGMCP-01CNSM-00016,,CGINS-MOPAK0-11843,40.13325,-70.778317,,0,135,,,
+AR66,,,,CP01CNSM-SBD11-02-HYDGN0000,16,1,2022-04-10T14:35:00,,CGMCP-01CNSM-00016,,CGINS-HYDGN0-42330,40.13325,-70.778317,,0,135,,,
+AR66,,,,CP01CNSM-SBD11-06-METBKA000,16,1,2022-04-10T14:35:00,,CGMCP-01CNSM-00016,,CGINS-METLGR-00010,40.13325,-70.778317,,0,135,,,
+AR66,,,,CP01CNSM-SBD12-00-DCLENG000,16,1,2022-04-10T14:35:00,,CGMCP-01CNSM-00016,,3718-00001-00001-00054,40.13325,-70.778317,,0,135,,,
+AR66,,,,CP01CNSM-SBD12-03-HYDGN0000,16,1,2022-04-10T14:35:00,,CGMCP-01CNSM-00016,,CGINS-HYDGN0-Z5971,40.13325,-70.778317,,0,135,,,
+AR66,,,,CP01CNSM-SBD12-04-PCO2AA000,16,1,2022-04-10T14:35:00,,CGMCP-01CNSM-00016,,CGINS-PCO2AA-31124,40.13325,-70.778317,,0,135,,,
+AR66,,,,CP01CNSM-SBD12-05-WAVSSA000,16,1,2022-04-10T14:35:00,,CGMCP-01CNSM-00016,,CGINS-WAVSSA-05781,40.13325,-70.778317,,0,135,,,
+AR66,,,,CP01CNSM-SBD12-06-METBKA000,16,1,2022-04-10T14:35:00,,CGMCP-01CNSM-00016,,CGINS-METLGR-00017,40.13325,-70.778317,,0,135,,,
+AR66,,,,CP01CNSM-SBD12-08-FDCHPA000,16,1,2022-04-10T14:35:00,,CGMCP-01CNSM-00016,,CGINS-FDCHPB-11802,40.13325,-70.778317,,-5,135,,,
```
Comment (cdobs) [inline deployment/CP01CNSM_Deploy.csv:545]: deployment time/location, recovery time, and deploy/water depth updates look good. configuration previously reviewed and approved.

File: deployment/CP03ISSM_Deploy.csv [hunk 2]
```diff
@@ -403,65 +403,65 @@ AR52,,AR61,,CP03ISSM-SBD11-06-METBKA000,13,1,2021-04-02T20:47:00,2021-11-11T19:3
...
+AR66,,,,CP03ISSM-SBC11-00-CPMENG000,15,1,2022-04-11T21:51:00,,CGMCP-03ISSM-00015,,3703-00051-00001-00099,40.367167,-70.881817,,0,94,CPM secondary 3703-00051-00001-01079,,
+AR66,,,,CP03ISSM-SBD11-00-DCLENG000,15,1,2022-04-11T21:51:00,,CGMCP-03ISSM-00015,,3718-00001-00001-00046,40.367167,-70.881817,,0,94,,,
+AR66,,,,CP03ISSM-SBD11-01-MOPAK0000,15,1,2022-04-11T21:51:00,,CGMCP-03ISSM-00015,,CGINS-MOPAK0-43257,40.367167,-70.881817,,0,94,,,
+AR66,,,,CP03ISSM-SBD11-02-HYDGN0000,15,1,2022-04-11T21:51:00,,CGMCP-03ISSM-00015,,CGINS-HYDGN0-42316,40.367167,-70.881817,,0,94,,,
+AR66,,,,CP03ISSM-SBD11-06-METBKA000,15,1,2022-04-11T21:51:00,,CGMCP-03ISSM-00015,,CGINS-METLGR-00008,40.367167,-70.881817,,0,94,,,
+AR66,,,,CP03ISSM-SBD12-00-DCLENG000,15,1,2022-04-11T21:51:00,,CGMCP-03ISSM-00015,,3718-00001-00001-00069,40.367167,-70.881817,,0,94,,,
+AR66,,,,CP03ISSM-SBD12-03-HYDGN0000,15,1,2022-04-11T21:51:00,,CGMCP-03ISSM-00015,,CGINS-HYDGN0-41917,40.367167,-70.881817,,0,94,,,
+AR66,,,,CP03ISSM-SBD12-04-PCO2AA000,15,1,2022-04-11T21:51:00,,CGMCP-03ISSM-00015,,CGINS-PCO2AA-35260,40.367167,-70.881817,,0,94,,,
```
Comment (cdobs) [inline deployment/CP03ISSM_Deploy.csv:467]: deployment time/location, recovery time, and deploy/water depth updates look good. configuration previously reviewed and approved.

File: deployment/CP04OSSM_Deploy.csv [hunk 3]
```diff
@@ -404,65 +404,65 @@ AR52,,AR61,,CP04OSSM-SBD12-00-DCLENG000,13,1,2021-03-30T15:02:00,2021-11-04T12:1
...
+AR66,,,,CP04OSSM-SBC11-00-CPMENG000,15,1,2022-04-09T14:56:00,,CGMCP-04OSSM-00015,,3703-00051-00001-01072,39.937517,-70.88685,,0,454,CPM secondary 3703-00051-00001-01073 ,,
+AR66,,,,CP04OSSM-SBD11-00-DCLENG000,15,1,2022-04-09T14:56:00,,CGMCP-04OSSM-00015,,3718-00001-00001-00014,39.937517,-70.88685,,0,454,,,
+AR66,,,,CP04OSSM-SBD11-01-MOPAK0000,15,1,2022-04-09T14:56:00,,CGMCP-04OSSM-00015,,CGINS-MOPAK0-42298,39.937517,-70.88685,,0,454,,,
+AR66,,,,CP04OSSM-SBD11-02-HYDGN0000,15,1,2022-04-09T14:56:00,,CGMCP-04OSSM-00015,,CGINS-HYDGN0-49886,39.937517,-70.88685,,0,454,,,
+AR66,,,,CP04OSSM-SBD11-06-METBKA000,15,1,2022-04-09T14:56:00,,CGMCP-04OSSM-00015,,CGINS-METLGR-00037,39.937517,-70.88685,,0,454,,,
+AR66,,,,CP04OSSM-SBD12-00-DCLENG000,15,1,2022-04-09T14:56:00,,CGMCP-04OSSM-00015,,3718-00001-00001-00051,39.937517,-70.88685,,0,454,,,
+AR66,,,,CP04OSSM-SBD12-03-HYDGN0000,15,1,2022-04-09T14:56:00,,CGMCP-04OSSM-00015,,CGINS-HYDGN0-49857,39.937517,-70.88685,,0,454,,,
+AR66,,,,CP04OSSM-SBD12-04-PCO2AA000,15,1,2022-04-09T14:56:00,,CGMCP-04OSSM-00015,,CGINS-PCO2AA-34233,39.937517,-70.88685,,0,454,,,
```
Comment (cdobs) [inline deployment/CP04OSSM_Deploy.csv:468]: deployment time/location, recovery time, and deploy/water depth updates look good. configuration previously reviewed and approved.

**Review by cdobs — APPROVED** [review]
Everything looks good

File: deployment/CP01CNSM_Deploy.csv (hunk 1 above)
Comment (IDCalTech) [reply deployment/CP01CNSM_Deploy.csv:545]: All updates look good. Approved.

File: deployment/CP03ISSM_Deploy.csv (hunk 2 above)
Comment (IDCalTech) [reply deployment/CP03ISSM_Deploy.csv:467]: All updates look good. Approved.

File: deployment/CP04OSSM_Deploy.csv (hunk 3 above)
Comment (IDCalTech) [reply deployment/CP04OSSM_Deploy.csv:468]: All updates look good. Approved.

**Review by IDCalTech — APPROVED** [review]
Looks good. Approved

**Pull request merged**